From 36b0b66f330bbef544caa17b22ac333e277e739c Mon Sep 17 00:00:00 2001
From: David Mitchell <davem@iabyn.com>
Date: Thu, 14 Apr 2016 15:39:37 +0100
Subject: [PATCH 1/9] Temporarily rename xsh_debug_log to su_debug_log

In a previous commit that function was renamed, but none of its callers
were renamed. For now, revert the function back to its old name
---
 xsh/debug.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xsh/debug.h b/xsh/debug.h
index 707c760..e44428c 100644
--- a/xsh/debug.h
+++ b/xsh/debug.h
@@ -9,7 +9,7 @@
 
 #if XSH_DEBUG
 # define XSH_D(X) STMT_START X STMT_END
-static void xsh_debug_log(const char *fmt, ...) {
+static void su_debug_log(const char *fmt, ...) {
  va_list va;
  SV *sv;
  dTHX;
-- 
2.8.3

